src/app/components/Tooltip/Tooltip.styles.ts

```diff
@@ -0,0 +1,8 @@
+export const getVariantClassNames = (variant: string) => {
+	switch (variant) {
+		case "small":
```

Comment:
@romanlikhovich nice work! If it's not causing problems in the existing PR, would you consider adding `sm` instead of `small` to follow the standards for all components?  https://github.com/ArkEcosystem/desktop-wallet/blob/be3a514a31376dc6da611add8cb54ca44f701c6d/src/types/index.ts#L3
Cheers

Reply:
@goga-m Thank you. I missed this case